src/client/gfs_client.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CLIENT_GFS_CLIENT_H
```

Comment:
let's define this as `GFS_CLIENT_GFS_CLIENT_H_`, since other PR has used `GFS` as leading prefix, with source directory paths for remaining header guard

Reply:
Resolved.